Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -0,0 +1,316 @@
+﻿using Kudu.Contracts.Tracing;
+using Kudu.Core.Infrastructure;
+using Kudu.Core.Tracing;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Kudu.Services.Performance
+{
+    internal static class ProfileManager
+    {
+        private const int ProcessExitTimeoutInSeconds = 300;
```

Comment:
I see, I was just curious as to why the timeout jumped from 20 to 300 seconds. It looks fine.